### DISCUSSION TITLE
Add a control to the Wizard?

### DISCUSSION THREAD

**Discussion by matthew-mayer** [dahall/TaskScheduler#922] [Q&A] [answered]
Hi, I was wondering if it's possible to add a control like an input box (for an end time) to the task scheduler wizard? If so, how would I do that? 

Thanks.

**Comment by dahall** [comment, marked as answer]
That is not possible at the moment. The wizard is an exact duplicate of the one from Microsoft's Task Scheduler manager which doesn't offer that.
